src/main/java/org/helpberkeley/memberdata/Main.java

```diff
@@ -880,14 +901,22 @@ private static String generateDriverPosts(
             statusMessages.append(url).append("\n");
         }
 
-        if (backupDriversPostURL != null) {
-            statusMessages.append("\n[Backup Drivers Message](").append(backupDriversPostURL).append(")");
+        if (backupDriversPostResponse != null) {
+            postIds.add(backupDriversPostResponse.postId);
+            statusMessages.append("\n[Backup Drivers Message](").append(backupDriversPostResponse.URL).append(")");
+        }
+
+        if (! topic.equals(Constants.TOPIC_DRIVERS_POST_STAGING)) {
+            response = apiClient.changePostOwner(topic.getId(), postIds, dispatcherUsername);
+            if (response.statusCode() != HTTP_OK) {
+                statusMessages.append("Failed changing post ownership to: " + dispatcherUsername);
```

Comment:
Honestly, I'm not even sure how this would fail, but we check the status code for the other POST calls. The user can't manually change the ownership, all they could do is try again or copy and repost them under their own name. Should I add that suggestion to the error message or remove it entirely?

Reply:
The general rule for good error messages is that they should say something about remediation.  Here, the user can't do anything about this but we are definitely interested.  So I suggest adding the following to it: "The messages have been successfully posted. Please notify the developers that there was a failure."  

Also make sure that we are logging the actual failure information at ERROR level, so that we can go track it down in the log on AWS.